SimPPS/RPDigiProducer/plugins/RPVFATSimulator.cc

```diff
@@ -0,0 +1,45 @@
+#include "SimPPS/RPDigiProducer/plugins/RPVFATSimulator.h"
+#include "Geometry/VeryForwardRPTopology/interface/RPTopology.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include <vector>
+#include "TRandom.h"
```

Comment:
@mundim @civanch why this?

Reply:
Sorry, it was left behind when moving to CLHEP random generators. Solved.